specification/inference/put_hugging_face/PutHuggingFaceRequest.ts

```diff
@@ -44,6 +44,16 @@ import { Id } from '@_types/common'
  * * `e5-small-v2`
  * * `multilingual-e5-base`
  * * `multilingual-e5-small`
+ *
+ * For Elastic's `rerank` task:
+ * The selected model must support the `sentence-ranking` task and expose OpenAI API.
+ * HuggingFace supports only dedicated (not serverless) endpoints for `Rerank` so far.
+ * After the endpoint is initialized, copy the full endpoint URL for use.
+ * Tested models for `rerank` task:
+ *
+ * * `bge-reranker-base`
```

Comment:
@szabosteve should we include the url to the model on hugging face? Or is the name sufficient here?

Reply:
We answered this in a previous thread. We're just going to use the names here.